RIOT-OS/RIOT
# drivers: add driver specific Makefile.dep

### Contribution description

This PR splits the main `drivers/Makefile.dep` in driver specific `Makefile.dep`s.

The dependency resolution for drivers modules defined as pseudo-modules still have to be done in the main `drivers/Makefile.dep` but just add the "real" module as a dependency.

There was a dependency resolution for `nrfmin` in `drivers/Makefile.dep` but it has nothing to do here, this PR moves it to `cpu/nrf5x_common/Makefile.dep`.




### Testing procedure

- A green Murdock
- Use build system tools to ensure no dependency has changed (help: what are the commands already ?)




### Issues/PRs references

Similar to #14369 but for Makefile.dep
